src/doc/contrib/src/implementation/debugging.md

```diff
@@ -8,7 +8,7 @@ It also supports filtering for specific modules with comma-separated [directives
 Feel free to use [shorthand macros] to help with diagnosing problems.
 We're looking forward to making Cargo logging mechanism more structural!
 
-```sh
+```bash
 # Outputs all logs with levels debug and higher
 CARGO_LOG=debug cargo generate-lockfile
 
```

Comment:
I tend to lean towards `console` blocks for commands

Reply:
Fair point as it feels shell-independent. Will fix those!

Reply:
I've tended to do `$ #`.  Not great but I feel its more semantically correct than describing these as bash scripts (which is what `bash` means).  Unsure if others have other opinions.

Reply:
That sounds fair, however for especially one-liner like this, I'd prefer the ease of copy-pasting by removing leading `$`: 

https://github.com/rust-lang/cargo/blob/f6cea42b7988fd737409abc7521a9ade796c26cf/src/doc/contrib/src/tests/writing.md?plain=1#L224-L226